.github/workflows/continuous-integration.yml

```
@@ -161,12 +217,27 @@ jobs:
           echo -e "\033[0;33mExecuting PHPUnit...\033[0m"
           php ../../vendor/bin/phpunit --colors=always
```

It is prone to conflicts, but having the same logic everywhere is preferable.

```suggestion
          if [[ -f "vendor/bin/phpunit" ]]; then
            vendor/bin/phpunit --colors=always
          elif [[ -f "../../vendor/bin/phpunit" ]]; then
            ../../vendor/bin/phpunit --colors=always
          else
            echo -e "\033[0;31mPHPUnit binary not found!\033[0m"
            exit 1
          fi
```